config/webpack.config.js

```diff
@@ -14,12 +16,15 @@ var production =
   process.env.RAILS_ENV === 'production' ||
   process.env.NODE_ENV === 'production';
 
-var config = {
-  entry: {
-    // Sources are expected to live in $app_root/webpack
-    bundle: './webpack/assets/javascripts/bundle.js'
-  },
+var plugins = JSON.parse(execSync('./script/plugin_webpack_directories.rb'));
 
+var config = {
+  entry: Object.assign(
+    {
+      bundle: './webpack/assets/javascripts/bundle.js'
+    },
+    plugins.entries
```

Comment:
I think "core" comes from plugins that have it in their names, e.g. foreman-tasks-core, rex-core etc.

have a look at the ruby script output, it defines the chunk name. I'm uncertain why you have the extra word core in there either.